doc/Language/5to6-perlfunc.pod6

```diff
@@ -1619,10 +1619,6 @@ X<| rewinddir - perlfunc>
 
 Not supported in Raku.
 
-The Raku ecosystem has a module L<C<P5rewinddir>|https://modules.raku.org/dist/P5rewinddir>
-which exports a C<rewinddir> function that mimics the original Perl
-behavior as much as possible.
-
```

Comment:
This is now part of [p5opendir](https://modules.raku.org/dist/P5opendir). Can you please change the link to point to that?

Reply:
@JJ changed to P5opendir as requested.